example/pubspec.yaml

```diff
@@ -11,6 +11,9 @@ dependencies:
   flutter:
     sdk: flutter
   provider:
+  flutter_driver:
+    sdk: flutter
+  test: ^1.15.7
```

Comment:
These should be dev dependencies

And `test` is not needed  because we already have `flutter_test`

Reply:
oh ok, Thanks @rrousselGit 
but we need `test: ^1.15.7` for actual test functions and assertions.

Reply:
I have added to dev dependencies.

Reply:
Did you push it? I still see the dependencies in `dependencies`

Reply:
Not yet.

Reply:
Done